examples/llava/mtmd-cli.cpp

```diff
@@ -24,7 +24,9 @@
 #include <signal.h>
 #endif
 
-static bool g_is_generating = false;
+//volatile, because of signal being an interrupt
+static volatile bool g_is_generating = false;
+static volatile bool is_app_running = true;
```

Comment:
```suggestion
static volatile bool g_is_interrupted = true;
```

I think the reverted name make more sense, it's easier to understand the code